packages/customize-widgets/src/components/inspector/style.scss

```diff
@@ -0,0 +1,27 @@
+@import "../../../../interface/src/components/complementary-area/style.scss";
```

Comment:
What's this for? It's odd to reach into a different package to import some CSS. Ideally we'd lean on component composition and e.g. import `@wordpress/interface` and use `<ComplementaryArea>`.

Reply:
It's for loading the complementary area styles to match the styles in the post editor more closely.

I think the pattern is used in [`edit-widgets`](https://github.com/WordPress/gutenberg/blob/65a6e7587b6d2c9439fc6e2a3adf58e7ed6a6a18/packages/edit-widgets/src/style.scss#L1) though 🤔?

I've tried using `<ComplementaryArea>` directly, but it's more complicated and contains a lot of unneeded code. At some point it requires changing the code in `<ComplementaryArea>` too.

Maybe we should just copy some of the CSS to `<BlockInspector>`.

Reply:
What feels icky to me is that doing this means that we will have `.interface-complementary-area` rules in `packages/customize-widgets/build-style/style.css`. It's confusing for one package's CSS to style another package's HTML.

> Maybe we should just copy some of the CSS to `<BlockInspector>`.

This sounds nicer!

Reply:
> I think the pattern is used in [`edit-widgets`](https://github.com/WordPress/gutenberg/blob/65a6e7587b6d2c9439fc6e2a3adf58e7ed6a6a18/packages/edit-widgets/src/style.scss#L1) though 🤔?

Ugh yes we should fix this too 😅

Reply:
Happy to do this in a follow-up if you'd prefer, let me know.

Reply:
I can (and I think it makes sense to) do this in this PR :)